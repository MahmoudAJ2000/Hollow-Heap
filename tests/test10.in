10 30 10 6
7 5 771
6 0 906
2 3 265
9 8 920
5 6 164
8 1 944
2 0 54
9 5 863
1 0 505
8 6 479
9 3 742
7 2 415
5 3 865
5 1 125
6 8 699
0 4 605
9 7 57
8 4 662
3 5 352
0 7 421
3 1 901
0 1 407
3 6 793
6 2 743
6 7 413
0 9 490
2 5 507
0 5 626
4 9 920
8 2 255
4
3
8
6
0
1
9
7
2
5
