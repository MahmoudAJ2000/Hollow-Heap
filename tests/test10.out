Distance from 6 to 4 is 1361
Distance from 6 to 3 is 1008
Distance from 6 to 8 is 699
Distance from 6 to 6 is 0
Distance from 6 to 0 is 797
Distance from 6 to 1 is 1204
Distance from 6 to 9 is 1287
Distance from 6 to 7 is 413
Distance from 6 to 2 is 743
Distance from 6 to 5 is 1184
